cogip/tools/detector/detector.py

```diff
@@ -76,60 +73,82 @@ def __init__(
             logger=True
         )
 
+        self._lidar_reader_loop = ThreadLoop(
+            "Lidar reader loop",
+            refresh_interval,
+            self.read_lidar,
+            logger=True
+        )
+
+        self._laser: ydlidar.CYdLidar | None = None
+        self._scan:  ydlidar.LaserScan | None = None
+        uart_port: str | None = None
+        if ydlidar and not emulation:
+            for _, value in ydlidar.lidarPortList().items():
+                uart_port = value
+        if uart_port:
+            self._laser = ydlidar.CYdLidar()
+            self._laser.setlidaropt(ydlidar.LidarPropSerialPort, uart_port)
+            self._laser.setlidaropt(ydlidar.LidarPropSerialBaudrate, 230400)
+            self._laser.setlidaropt(ydlidar.LidarPropLidarType, ydlidar.TYPE_TRIANGLE)
+            self._laser.setlidaropt(ydlidar.LidarPropDeviceType, ydlidar.YDLIDAR_TYPE_SERIAL)
+            self._laser.setlidaropt(ydlidar.LidarPropSingleChannel, False)
+            self._laser.setlidaropt(ydlidar.LidarPropSampleRate, 5)
+            self._laser.setlidaropt(ydlidar.LidarPropIntenstiy, True)
+            self._laser.setlidaropt(ydlidar.LidarPropScanFrequency, 5.0)
+            self._laser.setlidaropt(ydlidar.LidarPropMaxRange, self._max_distance/1000)
+            self._laser.setlidaropt(ydlidar.LidarPropMinRange, self._min_distance/1000)
+            self._laser.setlidaropt(ydlidar.LidarPropInverted, False)
```

Comment:
Properties seems diffrent here than in `lidar.py`.
Changes are not obvious.

Reply:
What is different?

Reply:
I dont know, that's the point ! Lets talk about this on discord. ;)